src/components/withConfirmationPopup/withConfirmationPopup.tsx

```diff
@@ -42,7 +42,14 @@ const PopupContent: FC<PopupContentProps> = ({ setOpen, onConfirm }) => {
     <div className={styles.popupContent}>
       <div>Are you sure?</div>
       <div>
-        <ActionButton size="small" type="accept" onClick={onConfirm} />
+        <ActionButton
+          size="small"
+          type="accept"
+          onClick={(): void => {
+            onConfirm?.();
```

Comment:
Are we expecting to have action buttons without onConfirm functions?  This seems like it will just dump errors and make it hard to debug.

Reply:
Agree - in the initial version when there was no implementation on any button it made sense to make it optional. At this point there shouldn't be any buttons that don't do anything.

Reply:
Made `onConfirm` a required prop.